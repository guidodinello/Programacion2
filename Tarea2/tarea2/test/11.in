# Implementar copiarSegmento
copiarSegmento 1 1
insertarSegmentoDespues 1 (3,2.4)
copiarSegmento 1 1
imprimirCadena
finalCadena
insertarSegmentoDespues 3 (5,3.4) (8,9.4) (9,2.6)
copiarSegmento 1 1
copiarSegmento 1 4
copiarSegmento 2 3
copiarSegmento 4 4
Fin
