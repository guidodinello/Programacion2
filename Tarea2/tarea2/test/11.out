1>#  Implementar copiarSegmento.
2>
3>Segmento insertado después de loc.
4>(3,2.40)
5>(3,2.40)
6>loc al final.
7>Segmento insertado después de loc.
8>(3,2.40)
9>(3,2.40)(5,3.40)(8,9.40)(9,2.60)
10>(5,3.40)(8,9.40)
11>(9,2.60)
12>Fin.
